chapter7/web-ui-example/pom.xml

```diff
@@ -123,8 +123,14 @@
 	</dependencies>
 
 	<build>
+		<extensions>
+      		<extension>
```

Comment:
I am against having these kind of checks. The code should be platform agnostic as possible.